modules/gui/src/com/haulmont/cuba/gui/components/actions/FilterApplyAction.java

```diff
@@ -21,7 +21,7 @@
 import com.haulmont.cuba.gui.components.ListComponent;
 
 /**
- * DEPRECATED. List action to apply current filter by refreshing the underlying datasource.
+ * @deprecated List action to apply current filter by refreshing the underlying datasource.
  */
 @Deprecated
 public class FilterApplyAction extends AbstractAction {

```

Comment:
Sorry, this file got here by an accident - I just marked this with deprecated annotation and used auto-formatting.